coordinator/bridge/src/test/java/io/stargate/bridge/service/BaseBridgeServiceTest.java

```diff
@@ -73,7 +72,7 @@ public class BaseBridgeServiceTest {
   protected @Mock Persistence persistence;
   protected @Mock AuthorizationService authorizationService;
 
-  protected Connection connection = spy(mock(Connection.class));
+  protected Connection connection = mock(Connection.class);
```

Comment:
the new version of Mockito errored with "IllegalArgument Please don't pass mock here. Spy is not allowed on mock"